azkaban-web-server/src/test/java/azkaban/imagemgmt/services/ImageTypeServiceImplTest.java

```diff
@@ -75,6 +78,20 @@ public void testCreateImageType() throws Exception {
     Assert.assertEquals(100, imageTypeId);
   }
 
+  @Test
+  public void testFindImageTypeWithOwnersByName() throws Exception {
+    final ImageType imageType = new ImageType();
+    when(this.imageTypeDao.getImageTypeWithOwnershipsByName(any(String.class))).thenReturn
+   ((Optional<ImageType>) Optional.of(imageType));
+    this.imageTypeService.findImageTypeWithOwnersByName("anyString");
+  }
+
+  @Test(expected = ImageMgmtException.class)
+  public void testFindImageTypeWithOwnersByNameFailsWithImageMgmtException() throws Exception {
+    when(this.imageTypeDao.getImageTypeWithOwnershipsByName(any(String.class))).thenReturn(Optional.empty());
+    this.imageTypeService.findImageTypeWithOwnersByName("anyString");
```

Comment:
This test is not useful. Is it possible to create some image types with owners and validate if you can get image type?

Reply:
My understanding is that we'd then be testing the logic behind the imageTypeDao.getImageTypeWithOwnershipsByName(String s) method. Does that sound right to you? I can certainly add those tests to the right package if you'd like! Thank you.

Reply:
I have added the tests I believe you wanted in ImageTypeDaoTest. Let me know if that works for you.